sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogManager.scala

```diff
@@ -44,8 +44,12 @@ class CatalogManager(
   import CatalogV2Util._
 
   private val catalogs = mutable.HashMap.empty[String, CatalogPlugin]
+  private val validNameFormat = "([\\w_]+)".r
```

Comment:
Can we reuse the validation logic if you refer to somewhere for consistency?

Reply:
@HyukjinKwon Thank you for taking a look at this PR firstly.  
Secondly, the validation logic for dbname/tablename is in [#L141](https://github.com/apache/spark/blob/0c34b9654187bae0638ed964efd974fc1888f46f/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala#L141). But the funcation is `private`.  Also, the catalog name just from the SparkConf. it is without parse and analysis phase.
At last. I think the verification of catalog name may be a bit strict. just do a simple check [#comments](https://github.com/apache/spark/pull/33339#discussion_r669457946). Because some testcase failed eg: `org.apache.spark.sql.parquet` as a catalog name but contains `.`